Mathlib/Topology/Compactness/Compact.lean

```diff
@@ -708,9 +715,10 @@ theorem isCompact_univ [h : CompactSpace X] : IsCompact (univ : Set X) :=
   h.isCompact_univ
 #align is_compact_univ isCompact_univ
 
-theorem cluster_point_of_compact [CompactSpace X] (f : Filter X) [NeBot f] : ∃ x, ClusterPt x f :=
+theorem exists_clusterPt_of_compactSpace [CompactSpace X] (f : Filter X) [NeBot f] :
+    ∃ x, ClusterPt x f :=
   by simpa using isCompact_univ (show f ≤ 𝓟 univ by simp)
-#align cluster_point_of_compact cluster_point_of_compact
+#align cluster_point_of_compact exists_clusterPt_of_compactSpace
```

Comment:
```suggestion
#align cluster_point_of_compact exists_clusterPt_of_compactSpace

@[deprecated] -- Since 28 January 2024
alias cluster_point_of_compact := exists_clusterPt_of_compactSpace
```